_inc/client/plans/single-product-backup.scss

```diff
@@ -72,3 +109,24 @@
 .plan-radio-button__label {
 	white-space: pre-wrap;
 }
+
+.single-product-backup__accented-card {
+	max-width: 518px;
+	min-height: 313px;
+	background-color: #fff;
+	border: solid 1px #e6e6e6;
+}
+
+.single-product-backup__accented-card__header {
+	padding: 15px;
+
+	h3 {
```

Comment:
Let's add a class name to this heading instead of relying on an actual tag name here.